website/docs/general/building-apisix.md

```diff
@@ -0,0 +1,258 @@
+---
+id: building-apisix
+title: Building APISIX
+keywords:
+  - API gateway
+  - APISIX
+  - Apache APISIX
+  - Code Contribution
+  - Building APISIX
+description: Guide for building and running APISIX locally for development.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+If you are looking to contribute to APISIX or setup a development environment, this guide is for you.
+
+If you are looking to install and run APISIX, check out the [Installation](/docs/apisix/how-to-build) docs.
```

Comment:
We'd better also update `how-to-build` doc name, as the current doc is how to build, the other one is `installation`.